packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -605,6 +605,10 @@ class AnimationController extends Animation<double>
   /// repeat. The [status] will be reported as [AnimationStatus.reverse] when
   /// the animation runs from [max] to [min].
   ///
+  /// Each run of the animation will have a duration of [period]. If [period] is not
```

Comment:
both times: "period" should be enclosed in ` instead of [ ] since it is just an argument to the function.:

```suggestion
  /// Each run of the animation will have a duration of `period`. If `period` is not
```

Reply:
Thanks @goderbauer. same error yet !